<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapGeneratorOption</class>
 <widget class="QWidget" name="MapGeneratorOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Option</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/cogwheel.png</normaloff>../icons/cogwheel.png</iconset>
  </property>
  <widget class="QComboBox" name="unit_ComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Meter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Centimeter</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Map unit</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="mapWidth_Input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="mapHeight_Input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Map Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Map_Height</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cellSpace_Input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell Spacing</string>
   </property>
  </widget>
  <widget class="QLabel" name="unit_label_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>92</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cm</string>
   </property>
  </widget>
  <widget class="QLabel" name="unit_label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>92</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cm</string>
   </property>
  </widget>
  <widget class="QLabel" name="unit_label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>142</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/check.png</normaloff>../icons/check.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/cross-mark.png</normaloff>../icons/cross-mark.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
